lib/completion.ts

```diff
@@ -146,6 +148,27 @@ export class Completion implements CompletionInstance {
         completions.push(...choices);
       }
     }
+
+    const positionalKeys =
```

Comment:
I think it would be worth pulling this logic into a private helper:

```
choicesFromPositionals
```

and then you could do? `completions = [...completions, choicesFromPositionals]` or something along those lines?

Reply:
I'll split choicesCompletions into choicesFromOptions and choicesFromPositionals